src/CodeGen_Xtensa.cpp

```diff
@@ -900,6 +978,29 @@ HALIDE_ALWAYS_INLINE void store_predicated<native_vector_u8, native_vector_i32_x
     }
 }
 
+template <>
```

Comment:
Not for this PR, but we should try to come with a way to generate these (more) automatically, because there is a lot of repetition.